rocketpy/EnvironmentAnalysis.py

```diff
@@ -616,10 +699,68 @@ def animate_wind_gust_distribution_over_average_day(self):
         """Animation of how the wind gust distribution varies throughout the day."""
         ...
 
-    # TODO: Implement
+    # TODO: Create tests
     def animate_wind_profile_over_average_day(self):
```

Comment:
Did you tried this function on a jupyter notebook to see how it will works? LIke it will print the gif inline?

Reply:
Yes, the animation does work on Jupyter notebooks, as long as you use a suitable backend, such as %matplotlib notebook or %matplotlib widget. The default %matplotlib inline does not show animations, as only pngs are displayed.

This StackOverflow post dives more into this: https://stackoverflow.com/questions/43445103/inline-animations-in-jupyter